biz.aQute.resolve/src/biz/aQute/resolve/DistroRepository.java

```diff
@@ -0,0 +1,21 @@
+package biz.aQute.resolve;
+
+import java.io.File;
+
+import org.osgi.resource.Resource;
+
+import aQute.bnd.osgi.Domain;
+import aQute.bnd.osgi.repository.ResourcesRepository;
+import aQute.bnd.osgi.resource.ResourceBuilder;
+
+public class DistroRepository extends ResourcesRepository {
```

Comment:
Now I am starting to understand what you're trying to do ... Sorry for being daft.

So why not then just all the org.osgi.framework.{packages|capabilities}.extra to work? I think the extra's are not processed in distro mode but maybe it makes sense to still do this?

This is as far as I can see identical to augmenting in this case? At least I do not see how a distro should require anything form the resolved set?